scripts/helpers/time_series_helpers.py

```diff
@@ -363,11 +368,44 @@ def forecast_ets(dataframe, start_date, end_date, seasonal=None, damped_trend=Fa
     model = ETSModel(
         dataframe,
         error="add",
-        trend="add",
+        trend=trend,
         seasonal=seasonal,
         damped_trend=damped_trend,
-        seasonal_periods=seasonal_periods)
+        lbh-huudo-morehelpers
+        seasonal_periods=seasonal_periods,
+    )
+
     fit = model.fit()
     pred = fit.get_prediction(start=start_date, end=end_date)
     df = pred.summary_frame(alpha=0.05)
     return df
+
+def holt_winters(dataframe,forecast_count,seasonal_periods=None,seasonal=None,use_boxcox = None,initialization_method="estimated"):
+    """
+
+            Args:
+                Dataframe (Dataframe): Dataframe containing training timeseries dataset. Must be equally spaced with Date in the Index
+                forecast_count (int): Amount of data points to forecast
+                seasonal (String): Trend Component model. Optional. "Add", "mul" or None (default)
+                seasonal_periods (int): The number of periods in a complete seasonal cycle for seasonal (Holt-Winters) models. For example, 4 for quarterly data with an annual cycle or 7 for daily data with a weekly cycle. Required if seasonal is not None.
+                use_boxcox (bool): None (default) or True
+                initialization_method (String):
+            Returns:
+                Forecast Results (Dataframe),
```

Comment:
I'm all for auto docstrings, but these aren't right / complete

- Dataframe in the inputs should be dataframe
- initialization_method needs a description
- Return should be pdf and needs a description